docs/reference/installation.rst

```diff
@@ -265,27 +265,21 @@ Now, add the new ``Application`` Bundle into the kernel:
         }
     }
 
-And configure ``FosUserBundle`` to use the newly generated ``User`` and ``Group``
-classes:
+And then register the ``Application`` namespace in the ``composer.json`` file under ``autoload/psr-4`` section:
```

Comment:
Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Reply:
@gregoire we should merge it like this, it’s is more benefit then negative impact or fix it by our own, but I am on a phone right now

Reply:
Can't, the docs build fails.

Reply:
saw it afterwards :(

Reply:
I am not familiar with reStructuredText but according to https://en.wikipedia.org/wiki/ReStructuredText#Literal_blocks it would work with only 2 colons.
Json is also fixed, thanks to @greg0ire

Reply:
In vim : `set texwidth=72`, then `gqq` on the offending line and it will be fixed. Can you please do it @colshrapnel ?

Reply:
You're right, it works:

![combined colons](https://screenshotscdn.firefoxusercontent.com/images/67e1177b-8c5c-4681-a779-b9413369eef7.png)